src/components/ArticleList.jsx

```diff
@@ -0,0 +1,29 @@
+import { useEffect, useState } from "react";
+import { getAllArticles } from "../utils/get";
+
+export default function ArticleList() {
+  const [articles, setArticles] = useState([]);
+
+  useEffect(() => {
+    getAllArticles().then((response) => {
+      console.log(response.data.articles, "<-- all articles response");
```

Comment:
You could add some loading state here to conditionally render something to tell a user the data is still being fetched while they are waiting ( for those on a slower internet connection ) 💡